### PR TITLE
Package mm.0.8.2

### PR DESCRIPTION
**Pull request by toots**
### `mm.0.8.2`
The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)

---
* Homepage: https://github.com/savonet/ocaml-mm
* Source repo: git+https://github.com/savonet/ocaml-mm.git
* Bug tracker: https://github.com/savonet/ocaml-mm/issues

---
:camel: Pull-request generated by opam-publish v2.2.0

### CODE REVIEW

**Comment by mseri** [comment]
There is a strange failure in the lower-bounds check:
```
The following actions will be performed:
=== remove 1 package
  - remove    csexp             1.5.1          [conflicts with dune]
=== downgrade 2 packages
  - downgrade dune              3.6.2 to 3.6.0 [required by mm]
  - downgrade dune-configurator 3.6.2 to 2.7.0 [required by mm]
=== recompile 1 package
  - recompile mm                0.8.2 (pinned)

<><> Processing actions <><><><><><><><><><><><><><><><><><><><><><><><><><><><>
-> retrieved dune.3.6.0  (cached)
-> retrieved dune-configurator.2.7.0  (cached)
-> removed   mm.0.8.2
-> removed   dune-configurator.3.6.2
-> removed   csexp.1.5.1
-> removed   dune.3.6.2
-> installed dune.3.6.0
-> installed dune-configurator.2.7.0
[ERROR] The compilation of mm.0.8.2 failed at "dune build -p mm -j 47 @install".

#=== ERROR while compiling mm.0.8.2 ===========================================#
# context              2.2.0~alpha~dev | linux/x86_64 | ocaml-base-compiler.5.0.0 | pinned(https://github.com/savonet/ocaml-mm/archive/refs/tags/v0.8.2.tar.gz)
# path                 ~/.opam/5.0/.opam-switch/build/mm.0.8.2
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p mm -j 47 @install
# exit-code            1
# env-file             ~/.opam/log/mm-7-0985f1.env
# output-file          ~/.opam/log/mm-7-0985f1.out
### output ###
# (cd _build/default && /home/opam/.opam/5.0/bin/ocamlc.opt -w -40 -g -bin-annot -I src/.mm_video.objs/byte -I /home/opam/.opam/5.0/lib/ocaml/unix -I src/.mm_audio.objs/byte -I src/.mm_base.objs/byte -I src/.mm_image.objs/byte -no-alias-deps -open Mm_video -o src/.mm_video.objs/byte/mm_video__Video.cmi -c -intf src/video.mli)
# File "src/video.mli", line 117, characters 23-49:
# 117 | module MakeCanvas (_ : Mm_image.Image.CanvasImage) : sig
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Alert deprecated: module Mm_image.Mm_image
# this module is shadowed
# File "src/video.mli", line 117, characters 23-49:
# 117 | module MakeCanvas (_ : Mm_image.Image.CanvasImage) : sig
#                              ^^^^^^^^^^^^^^^^^^^^^^^^^^
# Error: Unbound module Mm_image.Image
```
do you know why it could happen?

File: packages/mm/mm.0.8.2/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
+# This file is generated by dune, edit dune-project instead
+opam-version: "2.0"
+synopsis:
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "LGPL-2.1"
```
Comment (mseri) [inline packages/mm/mm.0.8.2/opam:7]:
```suggestion
license: "LGPL-2.1-only"
```
or
```suggestion
license: "LGPL-2.1-or-later"
```

**Comment by toots** [comment]
License fixed upstream. I'm not sure what the build error is about, this module is internal to the library. Seems like a bug with `dune` maybe?

**Comment by mseri** [comment]
Looks like a bug in 3.6.0

File: packages/mm/mm.0.8.2/opam [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  "The mm library contains high-level to create and manipulate multimedia streams (audio, video, MIDI)"
+maintainer: ["The Savonet Team <savonet-users@lists.sourceforge.net>"]
+authors: ["Samuel Mimram <smimram@gmail.com>"]
+license: "LGPL-2.1-only"
+homepage: "https://github.com/savonet/ocaml-mm"
+bug-reports: "https://github.com/savonet/ocaml-mm/issues"
+depends: [
+  "ocaml" {>= "4.08"}
+  "ocaml" {with-test & >= "4.12"}
+  "dune" {>= "3.6"}
```
Comment (mseri) [inline packages/mm/mm.0.8.2/opam:13]:
```suggestion
  "dune" {>= "3.6.1"}
```

**Comment by mseri** [comment]
Thanks

**Pull request merged**